apps/coscrad-frontend/src/utils/generic-components/presenters/generic-detail-presenter.tsx

```diff
@@ -0,0 +1,24 @@
+import { IBaseViewModel, IDetailQueryResult } from '@coscrad/api-interfaces';
+import { CommandPanel } from '../../../components/dynamicViews/commands';
+import { NotFound } from '../../../components/NotFound';
+import { NOT_FOUND } from '../../../store/slices/interfaces/maybe-loadable.interface';
+
+export const GenericDetailPresenter = ({
```

Comment:
Be sure to go back and use this for whatever resource detail views you have a hard-wired placeholder.

Reply:
Looks like this has been done for resources, but not non-resource aggregates, which don't yet use the same API.